src/game.zig

```diff
@@ -401,6 +402,12 @@ pub const Player = struct { // MARK: Player
 		return eyeCoyote;
 	}
 
+	pub fn getFovModifierBlocking() f32 {
+		mutex.lock();
+		defer mutex.unlock();
+		return fovMod;
+	}
+
```

Comment:
Why not just make fovMod atomic?

Reply:
Question: Why does it have to be atomic, when other, similar, variables (like eyeVel or eyePos) are not atomic either?

Reply:
If it's only ever used by the update and render functions, then you are right, it doesn't need to be thread safe.